### PR TITLE
Update to IDEA_o1_v03 vertex and ability to specify resolution per layer

### PR DESCRIPTION
**Pull request by armin-ilg**
BEGINRELEASENOTES
- Update to IDEA_o1_v03 vertex
- Specify spatial and time resolutions per layer

ENDRELEASENOTES

Related to https://github.com/HEP-FCC/FCC-config/pull/206 
@BrieucF : Shall I also directly include the silicon wrapper digitisation in the test?

### CODE REVIEW

**Comment by BrieucF** [comment]
Hi Armin, thanks. If ready, yes you can add it

File: VTXdigi/src/VTXdigitizer.cpp [hunk 1]
```diff
@@ -126,22 +138,20 @@ StatusCode VTXdigitizer::execute(const EventContext&) const {
 
     // Smear the hit in the local sensor coordinates
     double digiHitLocalPosition[3];
-    if (m_readoutName == "VTXIBCollection" ||
-        m_readoutName == "VTXOBCollection" ||
-        m_readoutName == "VertexBarrelCollection" ||
+    int iLayer = m_decoder->get(cellID, "layer");
```
Comment (BrieucF) [inline VTXdigi/src/VTXdigitizer.cpp:141]: Is this crashing if the field "layer" is not defined in the bit-field encoding? Shall we set the "layer" string key as a Gaudi proprty, check if this exists in the provided readout, and elegantly crash if it does not? Or is this already crashing nicely?

Comment (armin-ilg) [reply VTXdigi/src/VTXdigitizer.cpp:141]: No, this does not crash nicely. Declare as key Gaudi property like "  declareProperty("inputSimHits", m_input_sim_hits, "Input sim vertex hit collection name");" ?

Comment (armin-ilg) [reply VTXdigi/src/VTXdigitizer.cpp]: I've added a check and tested it, let me know if it's fine. I furthermore also added the silicon wrapper digitisation :-)

**Comment by BrieucF** [comment]
Thanks Armin! The failing test is not due to your PR, have to be investigated, possibly in another PR

**Comment by armin-ilg** [comment]
> Thanks Armin! The failing test is not due to your PR, have to be investigated, possibly in another PR

Ok, shall we merge this PR and the one in FCC-config then?

**Pull request merged**